client/src/telemetry/interactive-examples.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "react";
+import { useGleanClick } from "./glean-context";
+import { IEX_DOMAIN } from "../env";
+
+/**
+ * Forwards user interactions with interactive-examples to Glean.
+ */
+export function useInteractiveExamplesActionHandler() {
+  const gleanClick = useGleanClick();
+  useEffect(() => {
+    const listener = (event: MessageEvent) => {
+      if (
+        event.origin === IEX_DOMAIN &&
+        typeof event.data === "object" &&
+        event.data.type === "action"
+      ) {
+        gleanClick(`interactive-examples: ${event.data.source}`);
+      }
+    };
+
+    window.addEventListener("message", listener);
+
+    return () => window.removeEventListener("message", listener);
+  });
```

Comment:
I believe we want an empty dependency array to avoid unnecessarily adding/removing the event listener on every re-render.

Reply:
Good catch, resolved via 427d5bb558c565f316235d47c957a0552adac5aa.

The effect even has a dependency (`gleanClick`), even though it shouldn't usually change.

Reply:
@caugner